common/supplemental/windowsZones.xml

```diff
@@ -460,9 +466,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<mapZone other="Belarus Standard Time" territory="001" type="Europe/Minsk"/>
 			<mapZone other="Belarus Standard Time" territory="BY" type="Europe/Minsk"/>
 
-			<!-- (UTC+03:00) Moscow, St. Petersburg, Volgograd -->
+			<!-- (UTC+03:00) Moscow, St. Petersburg -->
```

Comment:
This display name change is correct.  However, the line containing the `"RU"` territory should not have been deleted, but rather edited to remove Volgograd, as in `type="Europe/Moscow Europe/Kirov"`.